pages/resources/audit-log.mdx

```diff
@@ -251,4 +251,5 @@ Returns an [audit log](#audit-log-object) object for the guild. Requires the `VI
 | after?       | snowflake | Get entries after this entry ID                               |
 | limit?       | integer   | Max number of entries to return (1-100, default 50)           |
 | user_id?     | snowflake | Get actions made by a specific user                           |
+| target_id?   | snowflake | Get actions affecting a specific user                         |
```

Comment:
Does it have to be a user ID, or can it be any resource (e.g. channel)

Reply:
Tested just now and it appears to be for every resource

Reply:
> Probably user ID as I can say from description

The description can be always wrong, so it's better to ask/test